### ISSUE TITLE
% not working

### ISSUE THREAD

**Issue by schroejahr2** [devromans/payment-fee#4]
Hi, if i set the fee to percentage as example 2, i always get a fixed sum of 2€ instead of 2% of the total value.

Any Ideas ? 

Also even if i set the extension to activated:no it still adds the fee if the fee for the payment method is configured.

Thanks !

**Comment by schroejahr2** [comment]
Sorry i just noticed you stated its working for 1.7. im using 1.9.2 maybe thats why

**Comment by devromans** [comment]
Hello, should work with Magento 1.9.2 to, will be fixed soon.

**Comment by schroejahr2** [comment]
Hi, any progress on this ?

**Comment by kirmalbrar** [comment]
Hi, if i set the fee to percentage its not working . let me know please how can i solve this .

Thanks

**Comment by Kocarik** [comment]
Hello,

Now still not working on Magento version 1.9.3 , i got 404 Error.

**Comment by kasper-agg** [comment]
Yeah, does not work here either: 404 when navigating to admin -> Payment Method Fee

**Comment by kasper-agg** [comment]
404 error should be fixed for Magento 1.9.x versions with following changes.

I'm currently diffing all my changes, but it works now. As far as I can tell, I only moved the acl configuration from `adminhtml.xml` to `config.xml` inside the `<adminhtml>` tag:

```
    <adminhtml>
<!--
...
-->
        <acl>
            <resources>
                <admin>
                    <children>
                        <system>
                            <children>
                                <config>
                                    <children>
                                        <paymentfee>
                                            <title>Payment Method Fee</title>
                                        </paymentfee>
                                    </children>
                                </config>
                            </children>
                        </system>
                    </children>
                </admin>
            </resources>
        </acl>
    </adminhtml>
```

After that you can safely delete `adminhtml.xml`. 
**note:** Don't forget to logout and login after your changes!

**Comment by Jeeva-Rathinam** [comment]
Anyone help me, how can i remove Rs. Symbol, because i am set the condition like %, but check out Page display Rs. symbol Customer gets confused. http://i.imgur.com/J0CNRrw.png

**Comment by rozzilla** [comment]
I've fixed with this commit: https://github.com/rozzilla/payment-fee/commit/fb7e5e8c0c0777002fd4c24fb41864e47a8c05d9

**Comment by rozzilla** [comment]
You can merge my pull request ;)
https://github.com/devromans/payment-fee/pull/16
